options.go

```diff
@@ -91,3 +92,13 @@ func WithIgnoredFields(fieldName ...string) Option {
 		return nil
 	}
 }
+
+// WithPgPlaceholders will use parameters placeholders that are compatible with
+// the postgres pg driver which requires a placeholder like $1 instead of ?.
+// See: https://pkg.go.dev/github.com/lib/pq
```

Comment:
Just checking that we're linking this library intentionally, because I think we use `jackc/pgx/v5` in Boundary? FWIW this library is also in "maintenance mode": https://github.com/lib/pq#status

Reply:
Add add `pgx` as well to the comment.  they both require the same placeholders.  ty!